navigation.py

```diff
@@ -192,6 +221,14 @@ def try_login(driver, username, password):
 
     wait_for_next_page(driver)
 
+def login(driver, username, password):
+    """ Like try_login but throws error if the user could not be logged in regularly."""
```

Comment:
"Tries to log in with the given credentials. If this fails, then an error is thrown."